src/schemas/json/project.json

```diff
@@ -315,6 +319,7 @@
 			"type": "string"
 		},
 		"frameworks": {
+            "description": "Target frameworks that will be built, and dependencies that are specific to the configuration.",
```

Comment:
Suggestion: "... are specific to <del>the configuration</del> the build of this project for that framework"